TodoApp.ConsoleApp/UI/TaskOperations.cs

```diff
@@ -91,17 +90,18 @@ static void ManipulateTask()
         static void TaskMaker()
         {
             string title = UserInput.ReadText("Enter a title: ", true);
-            string message = UserInput.ReadText("Enter a description: ", true);
-            int deadLine = int.Parse(UserInput.ReadText("How many days you need to finish the task?: ",true),provider);
+            string description = UserInput.ReadText("Enter a description: ", true);
+            string strDueDate = UserInput.ReadText("Enter dueDate: ", true);
+            DateTime duedate = DateTime.Parse(strDueDate, CultureInfo.InvariantCulture, DateTimeStyles.RoundtripKind);
```

Comment:
Create a new `ReadDate` method in `UserInput`.
It should contain just these two lines and return the DateTime.